Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -1217,7 +1217,11 @@ - (void)saveWebViewScrollOffset {
 
 - (void)articleDidLoad {
     dispatch_async(dispatch_get_main_queue(), ^{
-        self.navigationItem.title = self.article.displaytitle.wmf_stringByRemovingHTML;
+        NSString *navigationTitle = self.article.displaytitle.wmf_stringByRemovingHTML;
+        if ([navigationTitle length] > 15) {
+            navigationTitle = [WMFCommonStrings backTitle];
```

Comment:
If you `nil` this out instead, you can use the the system localized "Back" title